scripts/lib/linkCheckerIgnores.ts

```diff
@@ -58,26 +45,27 @@ const SHOULD_BE_FIXED: FilesToIgnores = {
   "docs/api/qiskit/qiskit.circuit.SwitchCaseOp.md": [
     "circuit#qiskit.circuit.CASE_DEFAULT",
   ],
-  "docs/api/qiskit/qiskit.pulse.library.GaussianSquareDrag.md": [
-    "qiskit.pulse.library.Drag_class.rst#qiskit.pulse.library.Drag",
-    "qiskit.pulse.library.Gaussian_class.rst#qiskit.pulse.library.Gaussian",
-  ],
   "docs/api/qiskit/utils.md": [
     "https://github.com/python-constraint/python-constraint%3E__",
     "#qiskit.utils.optionals.HAS_TESTTOOLS",
     "#qiskit.utils.optionals.HAS_GRAPHVIZ",
     "#qiskit.utils.optionals.HAS_PYDOT",
   ],
   "docs/api/qiskit-ibm-provider/qiskit_ibm_provider.IBMProvider.md": [
-    "https://auth.quantum.ibm.com/api",
+    "https://auth.quantum-computing.ibm.com/api",
```

Comment:
This was a bad change - the API docs still use `quantum-computing` despite the URL migration.